tests/providers/databricks/sensors/test_databricks.py

```diff
@@ -0,0 +1,172 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+import unittest
+from unittest import mock
+from airflow.providers.databricks.sensors.databricks import  \
+    (DatabricksDeltaTableChangeSensor, DatabricksPartitionTableSensor)
+from databricks.sql.types import Row
+
+
+DEFAULT_CONN_ID = 'databricks_default'
+
+
+class TestDatabricksDeltaTableChangeSensor(unittest.TestCase):
+
+    @mock.patch('airflow.providers.databricks.sensors.databricks.DatabricksSqlHook')
+    def test_exec_success_sensor_find_new_events(self, db_mock_class):
+        """
+        Test the sensor when find a new event from the Databricks Delta Table
+        """
+
+        sensor = DatabricksDeltaTableChangeSensor(
+            task_id='run_now',
+            databricks_conn_id='databricks_default',
+            table='table_test',
+            timestamp='2022-05-07 21:24:58.680358'
```

Comment:
Consider making the timestamp string a global so you can also reuse it in lines 64, 77, and 100

Reply:
Fixed